core/controller/src/main/scala/whisk/core/entitlement/RateThrottler.scala

```diff
@@ -43,14 +44,17 @@ class RateThrottler(description: String, defaultMaxPerMinute: Int, overrideMaxPe
   /**
    * Checks whether the operation should be allowed to proceed.
    * Every `check` operation charges the subject namespace for one operation.
+   * Allows 20% of additional requests on top of the limit to mitigate possible unfair round-robin loadbalancing between
+   * controllers
    *
    * @param user the identity to check
    * @return true iff subject namespace is below allowed limit
    */
   def check(user: Identity)(implicit transid: TransactionId): RateLimit = {
     val uuid = user.uuid // this is namespace identifier
     val throttle = rateMap.getOrElseUpdate(uuid, new RateInfo)
-    val limit = overrideMaxPerMinute(user).getOrElse(defaultMaxPerMinute)
+    val limit =
+      Math.ceil((overrideMaxPerMinute(user).getOrElse(defaultMaxPerMinute).toDouble / diviser.toDouble) * 1.2).toInt
```

Comment:
This computation needs to be done only iff controllerHA is active. I propose to adjust the values passed into the RateThrottlers based on the controllerHA value one level above instead of passing everything through here. Should leave the RateThrottler untouched.